src/components/project/ProjectPreview.js

```diff
@@ -1,29 +1,31 @@
 import React from "react";
 import Router from "next/router";
-import {
-  Typography,
-  Card,
-  CardActionArea,
-  CardMedia,
-  CardContent,
-  List,
-  ListItem,
-  ListItemIcon,
-  ListItemText
-} from "@material-ui/core";
-import PlaceIcon from "@material-ui/icons/Place";
-import BarChartIcon from "@material-ui/icons/BarChart";
-import GroupIcon from "@material-ui/icons/Group";
+import { Typography, Card, CardMedia, CardContent } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
+import ProjectMetaData from "./ProjectMetaData";
 
-const useStyles = makeStyles({
-  root: {
-    display: "flex"
-  },
-  media: {
-    width: 250,
-    backgroundSize: "cover"
-  }
+const useStyles = makeStyles(theme => {
+  return {
+    root: {
+      "&:hover": {
+        cursor: "pointer"
+      },
+      "-webkit-user-select": "none",
+      "-moz-user-select": "none",
+      "-ms-user-select": "none",
+      userSelect: "none",
+      backgroundColor: "#fafafa",
```

Comment:
Should this color come from a theme?

Reply:
That was the color where I had trouble accessing it and you suggested it might be in CssBaseline. I changed it to backgroundColor: "inherit" which resolves to the same color.